Packs/Tripwire/Integrations/Tripwire/Tripwire.py

```diff
@@ -0,0 +1,469 @@
+    if args.get('time_detected_range'):
+        filters += f"timeDetectedRange={args.get('time_detected_range')}&"
+    if args.get('time_received_range'):
+        filters += f"timeReceivedRange={args.get('time_received_range')}&"
+    if args.get('limit'):
+        filters += f"pageLimit={args.get('limit')}&"
+    if args.get('start'):
+        filters += f"pageStart={args.get('start')}&"
+    return filters
+
+
+def filter_rules(args: dict) -> str:
+    """
+    :param args: (dict) args of the command .
+    :return: (str) combination of all filter for the command.
+    """
+    filters = ""
+    if args.get('rule_oids'):
+        filters += f"id={args.get('rule_oids')}&"
+    if args.get('rule_names'):
+        filters += f"name={args.get('rule_names')}&"
+    if args.get('rule_types'):
+        filters += f"type={args.get('rule_types')}&"
+    if args.get('limit'):
+        filters += f"pageLimit={args.get('limit')}&"
+    if args.get('start'):
+        filters += f"pageStart={args.get('start')}&"
+    return filters
+
+
+def filter_elements(args: dict) -> str:
+    """
+    :param args: (dict) args of the command .
+    :return: (str) combination of all filter for the command.
+    """
+    filters = ""
+    if args.get('element_oids'):
+        filters += f"id={args.get('element_oids')}&"
+    if args.get('element_names'):
+        filters += f"name={args.get('element_names')}&"
+    if args.get('node_oids'):
+        filters += f"nodeId={args.get('node_oids')}&"
+    if args.get('rule_oids'):
+        filters += f"ruleId={args.get('rule_oids')}&"
+    if args.get('baseline_version_ids'):
+        filters += f"baselineVersionId={args.get('baseline_version_ids')}&"
+    if args.get('last_version_id'):
+        filters += f"lastVersionId={args.get('last_version_id')}&"
+    if args.get('limit'):
+        filters += f"pageLimit={args.get('limit')}&"
+    if args.get('start'):
+        filters += f"pageStart={args.get('start')}&"
+    return filters
+
+
+def filter_nodes(args: dict) -> str:
+    """
+    :param args: (dict) args of the command .
+    :return: (str) combination of all filter for the command.
+    """
+    filters = ""
+    if args.get('node_oids'):
+        filters += f"id={args.get('node_oids')}&"
+    if args.get('node_ips'):
+        filters += f"ipAddress={args.get('node_ips')}&"
+    if args.get('node_mac_adresses'):
+        filters += f"macAddress={args.get('node_mac_adresses')}&"
+    if args.get('node_names'):
+        filters += f"ic_name={args.get('node_names')}&"
+    if args.get('node_os_names'):
+        filters += f"make={args.get('node_os_names')}&"
+    if args.get('tags'):
+        filters += f"tag={args.get('tags')}&"
+    if args.get('limit'):
+        filters += f"pageLimit={args.get('limit')}&"
+    if args.get('start'):
+        filters += f"pageStart={args.get('start')}&"
+    return filters
+
+
+def prepare_fetch(params: dict, first_fetch: str):
```

Comment:
if used only from the `fetch_incidents` command, why not call it from within there?
i think it will also allow you to return just the necessary param and not all of them. WDYT?